<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectYourTour</class>
 <widget class="QDialog" name="SelectYourTour">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>192</width>
    <height>95</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>65</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QPushButton" name="doneButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>65</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QComboBox" name="nameDropdown">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>35</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="selectLabel">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>3</y>
     <width>190</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Select a Stadium You
 Would Like to Visit:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="specifiedButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>35</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Order 
Specified</string>
   </property>
  </widget>
  <widget class="QPushButton" name="efficientButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>35</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Most Efficient 
Order</string>
   </property>
  </widget>
  <widget class="QLabel" name="orderLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Which Order Would You 
Like to Perform The Tour?</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
